cmake/onnxruntime_providers_tensorrt.cmake

```diff
@@ -73,11 +132,11 @@
       unset(PROTOBUF_LIBRARY)
       unset(OLD_CMAKE_CXX_FLAGS)
       unset(OLD_CMAKE_CUDA_FLAGS)
-      set_target_properties(nvonnxparser PROPERTIES LINK_FLAGS "/ignore:4199")
-      target_compile_options(nvonnxparser_static PRIVATE /FIio.h /wd4100)
-      target_compile_options(nvonnxparser PRIVATE /FIio.h /wd4100)
+      set_target_properties(${PARSER_LIB} PROPERTIES LINK_FLAGS "/ignore:4199")
+      target_compile_options(${PARSER_LIB}_static PRIVATE /FIio.h /wd4100)
+      target_compile_options(${PARSER_LIB} PRIVATE /FIio.h /wd4100)
     endif()
-    set(onnxparser_link_libs nvonnxparser_static)
+    set(onnxparser_link_libs ${PARSER_LIB}_static)
```

Comment:
@kevinch-nv would the static lib built using oss parser have the version in it as well? i.e. nvonnxparser_10_static.lib ? that feels a bit unconventional.

Reply:
Good catch. Static libraries are just `nvonnxparser_static` on all platforms.